java/code/src/com/redhat/rhn/common/util/FileUtils.java

```diff
@@ -130,27 +113,19 @@ public static String readStringFromFile(String path) {
      * @return String containing file.
      */
     public static String readStringFromFile(String path, boolean noLog) {
-        if (log.isDebugEnabled()) {
-            log.debug("readStringFromFile: {}", StringUtil.sanitizeLogInput(path));
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("readStringFromFile: {}", StringUtil.sanitizeLogInput(path));
         }
 
-        File f = new File(path);
-        BufferedReader input;
         try {
-            input = new BufferedReader(new FileReader(f));
-            StringWriter writer = new StringWriter();
-            IOUtils.getInstance().copyWriter(input, writer);
-            String contents = writer.toString();
-            if (noLog && log.isDebugEnabled()) {
-                log.debug("contents: {}", contents);
+            String contents = Files.readString(Path.of(path));
+            if (noLog && LOGGER.isDebugEnabled()) {
```

Comment:
I keept this how it was before but, shouldn't this be `!noLog && LOGGER.isDebugEnabled()` or am I missing something?

Reply:
indeed that rings a bell and I though @mcalmer fixed it already...

Reply:
Ahh, this is in a WIP PR which will take longer. I would suggest to fix it here again as it will be merged earlier than mine.

Reply:
Fixed